main.py

```diff
@@ -2,6 +2,34 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    genres_list = ["Western", "Action", "Dramma"]
+    for genre in genres_list:
+        Genre.objects.create(name=genre)
```

Comment:
it's better to add blank lines before loops and conditions, to increase readability